libraries/AC_AutoTune/AC_AutoTune.h

```diff
@@ -32,13 +71,15 @@ class AC_AutoTune {
     virtual void run();
 
     // save gained, called on disarm
-    void save_tuning_gains();
+    // made virtual for to add multi/heli specific save routines
+    virtual void save_tuning_gains();
 
     // stop tune, reverting gains
     void stop();
 
     // reset Autotune so that gains are not saved again and autotune can be run again.
-    void reset() {
+    void reset()
```

Comment:
no need to move the "{" to the next line in a .h file.

Reply:
done. Thanks!